Cargo.toml

```diff
@@ -1,20 +1,15 @@
 [package]
 name = "tree-sitter-smithy"
 description = "smithy grammar for the tree-sitter parsing library"
-version = "0.0.1"
+version = "1.0.0"
 keywords = ["incremental", "parsing", "smithy"]
 categories = ["parsing", "text-editors"]
 repository = "https://github.com/tree-sitter/tree-sitter-smithy"
```

Comment:
While you are here, would you mind please updating the repository URL?
Not that it's used anywhere, but still :)

Reply:
sure sure